### DISCUSSION TITLE
Running topic-operator alone with external kafka cluster

### DISCUSSION THREAD

**Discussion by noorul** [strimzi/strimzi-kafka-operator#4897] [General]
We decided to use the cloud provider-hosted Kafka service. Is it possible to use the topic operator alone with an externally hosted Kafka cluster?

**Comment by scholzj** [comment]
You can do it in general: https://strimzi.io/docs/operators/latest/full/deploying.html#deploying-the-topic-operator-standalone-str ... but there are some limitations as to what authentication it supports etc.
